src/components/editor/editor-pane/tool-bar/editor-preferences/editor-preference-ligatures-select.tsx

```diff
@@ -21,8 +21,8 @@ export const EditorPreferenceLigaturesSelect: React.FC = () => {
   return (
     <EditorPreferenceInput onChange={saveLigatures} value={ligaturesEnabled} property={"ligatures"}
                            type={EditorPreferenceInputType.BOOLEAN}>
-      <option value='false'>{t(`editor.modal.preferences.ligatures.off`)}</option>
-      <option value='true'>{t(`editor.modal.preferences.ligatures.on`)}</option>
+      <option value='false'>{t(`common.off`)}</option>
+      <option value='true'>{t(`common.on`)}</option>
```

Comment:
Better but this time you're refering to `common.on` instead of `common.yes`. For no/off similar.

Reply:
changed